docker/scripts/app-init.sh

```diff
@@ -4,6 +4,12 @@ WORKSPACEDIR="/root/src"
 
 service rsyslog start
 
+# Generate static assets
+
+npm install
+echo "Building static assets... (this could take a minute or two)"
+npx parcel build
```

Comment:
Should maybe explain that `npx parcel build` must be re-run when sources are changed, since this is a new step with bs5?